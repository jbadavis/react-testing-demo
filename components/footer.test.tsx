import {render} from '@testing-library/react';
import Footer from './footer';

describe('Footer', () => {
  test('renders correctly', () => {
    const {container} = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
