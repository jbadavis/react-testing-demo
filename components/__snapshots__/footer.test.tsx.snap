// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <div
    class="css-rayotx"
  >
    <div
      class="css-azx95j"
    >
      <h4
        class="chakra-heading css-1157erc"
      >
        Testing Demonstration ðŸ‘»
      </h4>
    </div>
  </div>
</div>
`;
